app/routes/circulars.moderation.$circularId.$requestor.tsx

```diff
@@ -9,6 +9,9 @@ import type { SEOHandle } from '@nasa-gcn/remix-seo'
 import type { ActionFunctionArgs, LoaderFunctionArgs } from '@remix-run/node'
 import { Form, redirect, useLoaderData } from '@remix-run/react'
 import { Button, ButtonGroup } from '@trussworks/react-uswds'
+import dayjs from 'dayjs'
+import locale from 'dayjs/locale/en'
+import RelativeTime from 'dayjs/plugin/relativeTime'
```

Comment:
This and the following code:

```ts
dayjs.locale(locale)
dayjs.extend(RelativeTime)
```

are duplicated from TimeAgo.tsx. Please refactor.